distributions/aws/examples/storage/fsx-for-lustre/README.md

```diff
@@ -0,0 +1,183 @@
+# Using Amazon FSx as Persistent Storage with Kubeflow
+
+This guide describes how to use Amazon FSx as Persistent storage on top of an existing Kubeflow deployment.  
+
+## 1.0 Prerequisites
+1. For this README, we will assume that you already have an EKS Cluster with Kubeflow installed since the FSx CSI Driver can be installed and configured as a separate resource on top of an existing Kubeflow deployment. You can follow any of the other guides to complete these steps - choose one of the [AWS managed service integrated offering](../../README.md) or [generic distribution](../../../../../README.md).
+
+2. At this point, you have likely cloned this repo and checked out the right branch. All paths in the following steps are relative to `kubeflow-manifests/distributions/aws/examples/storage` directory.
+
+3. Make sure the following environment variables are set. 
+```
+export CLUSTER_NAME=<clustername>
+export CLUSTER_REGION=<clusterregion>
+export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+```
+
+## 2.0 Install the FSx CSI Driver
+We recommend installing the FSx CSI Driver v1.3.4 directly from the [the aws-fsx-csi-driver github repo](https://github.com/kubernetes-sigs/aws-fsx-csi-driver) as follows - 
+
+```
+kubectl apply -k "github.com/kubernetes-sigs/aws-fsx-csi-driver/deploy/kubernetes/overlays/stable/?ref=tags/v0.7.1"
+```
+
+You can confirm that FSx CSI Driver was installed using the following command - 
+```
+kubectl get csidriver -A
+
+NAME              ATTACHREQUIRED   PODINFOONMOUNT   MODES        AGE
+fsx.csi.aws.com   false            false            Persistent   14s
+```
+
+## 3.0 Create the IAM Policy for the CSI Driver
+The CSI driver's service account (created during installation) requires IAM permission to make calls to AWS APIs on your behalf. Here, we will be annotating the Service Account `fsx-csi-controller-sa` with an IAM Role which has the required permissions.
+
+1. Create the policy using the json file provided as follows - 
+```
+aws iam create-policy \
+    --policy-name Amazon_FSx_Lustre_CSI_Driver \
+    --policy-document file://fsx-for-lustre/fsx-csi-driver.json
+```
+
+2. Create an IAM role and attach the IAM policy to it. Annotate the Kubernetes service account with the IAM role ARN and the IAM role with the Kubernetes service account name. You can create the role using eksctl as follows - 
+
+```
+eksctl create iamserviceaccount \
+    --name fsx-csi-controller-sa \
+    --namespace kube-system \
+    --cluster $CLUSTER_NAME \
+    --attach-policy-arn arn:aws:iam::$AWS_ACCOUNT_ID:policy/Amazon_FSx_Lustre_CSI_Driver \
+    --region $CLUSTER_REGION \
+    --approve \
+    --override-existing-serviceaccounts 
+```
+
+3. You can verify by describing the specified service account to check if it has been correctly annotated - 
```

Comment:
what field and value user need to look for?

Reply:
I think it is kind of obvious from the description in point 2 and the word `annotated` in point 3 that we are making sure the `annotations` field has a IAM Role attached. If you think it is confusing I can reword it but it is quite specific IMO.